docs/getting-started.md

```diff
@@ -77,8 +77,15 @@ If you want to add Windows Nodes to your cluster, please refer to these
 
 ### ARM support
 
-Starting with v0.13, Antrea supports arm64 and arm/v7 Nodes. Please refer to
-these [installation instructions](arm-support.md).
+Starting with v1.0, Antrea supports arm64 and arm/v7 Nodes. The installation
+instructions do not change when some (or all) Linux Nodes in a cluster use an
+ARM architecture: the `antrea/antrea-ubuntu` Docker image is actually a manifest
```

Comment:
It takes me some seconds to understand the sentence. Could we add more contexts here, like: the same Antrea deployment YAML can be used, etc.?

Reply:
Changed it a bit, let me know if you have something different in mind

Reply:
Works for me.